pkg/resource/testdata/TestStatefulSetBuilder/insecure_statefulset_cli_args.golden

```diff
@@ -78,15 +78,34 @@ spec:
         volumeMounts:
         - mountPath: /cockroach/cockroach-data/
           name: datadir
+      securityContext:
+        fsGroup: 10001
+        runAsUser: 10001
       serviceAccountName: cockroach-database-sa
       terminationGracePeriodSeconds: 60
       volumes:
       - name: datadir
         persistentVolumeClaim:
           claimName: ""
         name: datadir
+      initContainers:
+        - command:
+          - /bin/sh
+          - -c
+          - '>- cp -p /cockroach/cockroach-certs-prestage/..data/* /cockroach/cockroach-certs/
+            && chmod 700 /cockroach/cockroach-certs/*.key && chown 10001:10001 /cockroach/cockroach-certs/*.key'
+          image: registry.access.redhat.com/ubi8/ubi
```

Comment:
we need to use an already bundled image for the red hat offline process if possible.